content/blog/2021-06-10-on-ransomware-naming.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "On Ransomware Naming"
+created_at: 2021-06-10
+kind: article
+author_name: Richard "RichiH" Hartmann
+---
+
+As per Oscar Wilde, imitation is the sincerest form of flattery.
+
+The names "Prometheus" and "Thanos" have recently been taken up by a ransomware group. There's not much we can do about that except to inform you that this is happening. There's not much you can do either, except following the common supply chain & security practices. When deploying software, do it through one of those mechanisms:
+
+* Binary downloads from the official release pages for [Prometheus](https://github.com/prometheus/prometheus/releases) and [Thanos](https://github.com/thanos-io/thanos/releases), with verification of checksums provided.
+* Docker downloads from official project controlled repositories:
+  * Prometheus: https://quay.io/repository/prometheus/prometheus 
+  * Thanos: https://quay.io/repository/thanos/thanos and https://hub.docker.com/r/thanosio/thanos
+* Binaries, images, or containers from distributions you trust
+* Binaries, images, or containers from your own internal software verification and deployment teams
+* Build from source yourself
+
+Unless you can reasonably trust the specific providence and supply chain, you should not use software.
```

Comment:
While all this is great, as a reader I would immediately assume that there's a group out there that specifically tries to make people download trojanized versions of our Prometheus and Thanos, which is not the case at all. It could be good to point out that the group's name is "Prometheus" and the ransomware they use happens to be called "Thanos", but that we're not aware of any intended connection to us?

Reply:
Fair, will add.